src/lualib/Set.ts

```diff
@@ -6,6 +6,12 @@ Set = class Set<T> {
     private items: { [key: string]: boolean } = {};
```

Comment:
Does `Set` even need `items` anymore? `has` could just check `nextKey` or `previousKey`, couldn't it?

Reply:
Wait nevermind... I just realized that won't work for a one-item set.

Reply:
You do have a point, I think checking nextKey and lastKey would be enough to get rid of items and would reduce storage space required.

Reply:
I made this change, seems fine.